src/main.cpp

```diff
@@ -4752,7 +4752,8 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
                     LogPrintf("Force relaying tx %s from whitelisted peer=%d\n", tx.GetHash().ToString(), pfrom->id);
                     RelayTransaction(tx);
                 } else {
-                    LogPrintf("Not relaying invalid transaction %s from whitelisted peer=%d (%s)\n", tx.GetHash().ToString(), pfrom->id, FormatStateMessage(state));
+                    LogPrintf("Not relaying invalid transaction %s from whitelisted peer=%d (%s (code %i))\n",
```

Comment:
Why ``%i`` rather than ``%d``, incidentally? They're equivalent, both [in tinyformat](https://github.com/c42f/tinyformat/blob/master/tinyformat.h#L697) (which is what ``LogPrintf`` uses) and in C, but I see ``%i`` so rarely that I had to look it up.